### DISCUSSION TITLE
The synchronization of my documents to an empty repository is showing an error.

### DISCUSSION THREAD

**Discussion by BlackWhite2000** [GitbookIO/community#396] [3. Git Sync] [answered]
![image](https://github.com/GitbookIO/community/assets/39222378/3c6ff9b8-e22e-4456-9780-ed1433dbf4be)
![image](https://github.com/GitbookIO/community/assets/39222378/95bd7b85-b4e0-4e54-b8bb-a1a60e393f99)

**Comment by BlackWhite2000** [comment]
This issue seems to have arisen recently, as just a few days ago, I performed the same operation and it was successful. 
At that time, the synchronization UI on GitHub was not like it is now; the operation was done through the sidebar.

**Comment by BlackWhite2000** [comment, marked as answer]
I think I understand the issue now. 
The repository cannot be an empty repository; at least one README.md is required. 
However, I find this change to be unfriendly because the documentation creation starts from GitBook, not GitHub. 
In other words, it is inevitable to have an empty GitHub repository before pushing GitBook to GitHub

**Reply by aikoii** [reply]
Hey @BlackWhite2000, thank you for sharing your feedback! We'll look into improving the experience with empty repositories 🙂👍
